src/components/organisms/about/about.scss

```
@@ -0,0 +1,24 @@
+@use "../../variables" as *;
+.about {
+  padding: 100px 0;
+  @media screen and(max-width:$breakpoint-lg) {
+    padding: 40px;
+  }

```

```suggestion
  padding-top: 100px;
  padding-bottom: 100px;
  @media screen and(max-width:$breakpoint-lg) {
    padding-top: 40px;
    padding-bottom: 40px;
  }
```

すみません、 このような書き方の方が適切です🙏
`container` にはデフォルトで左右の `padding` がついているのですが、4方向を一度に指定するショートハンドを使うとそれを上書きしてしまうためです。